docs/framework/20categories/20architecture/messaging.md

```diff
@@ -37,6 +37,19 @@ While the above approaches offer better security guarantees because they remove
   * What is the cost of bribing or corrupting a threshold of such validators to violate safety or liveness?
   * How does the bridge adapt to active misbehavior by a portion of the validators?
 
-* Optimistic approaches - honest minority (but incentive alignment that such a minority exists)
+* Optimistic approaches: rely on agents that validate cross-chain transactions by signing a merkle root with data from the source network and post it on the destination network. Once this data is posted, a challenge window begins, during which anyone monitoring the bridging system can challenge a fraudulent transaction by submitting fraud proofs on the source network and prevent it from being confirmed on the destination network. Such bridges assume that a) agents are incentivized to sign only legitimate transactions because their bonded funds will be slashed if they act maliciously and b) that in the event an agent does sign a fraudulent transaction, one honest actor will be watching the system, and they will flag the fraudulent transaction by submitting a fraud-proof on the source network within the challenge window. Thus, optimistic bridges have a 1 of N security model, which relies on one honest actor watching the system to correctly verify cross-chain transactions.
```

Comment:
I think that it should be added the many of the optimistic systems used permissioned watchers only. 

Another thing to add is, once the fraud has been highlighted, the other agents that have been observing the source chain need to multiply sign to say that the Merkle Root on the destination chain is invalid, so that the fraudulent Merkle Root isn't acted upon.

Reply:
ok, permissioned watchers is mentioned in the considerations, so probably doesn't need to be mentioned in the paragraph above.